dataland-frontend/src/utils/DataOwnerUtils.ts

```diff
@@ -0,0 +1,33 @@
+import { assertDefined } from "@/utils/TypeScriptUtils";
+import type Keycloak from "keycloak-js";
+import { ApiClientProvider } from "@/services/ApiClients";
+import { type AxiosError } from "axios";
+import { waitForAndReturnResolvedKeycloakPromise } from "@/utils/KeycloakUtils";
+
+/**
+ * Check if a user is data owner of a company
+ * @param companyId the dataland companyId of the company for which ownership should be checked
+ * @param keycloakPromiseGetter the getter-function which returns a Keycloak-Promise
+ * @returns a promise, which resolves to a boolean
+ */
+export async function isUserDataOwnerForCompany(
+  companyId?: string,
+  keycloakPromiseGetter?: () => Promise<Keycloak>,
+): Promise<boolean> {
+  if (keycloakPromiseGetter && companyId) {
+    const resolvedKeycloakPromise = await waitForAndReturnResolvedKeycloakPromise(keycloakPromiseGetter);
+    const userId = resolvedKeycloakPromise?.idTokenParsed?.sub;
+    try {
+      await new ApiClientProvider(
+        assertDefined(keycloakPromiseGetter)(),
+      ).backendClients.companyDataController.isUserDataOwnerForCompany(companyId, assertDefined(userId));
+      return true;
+    } catch (error) {
+      console.log(error);
```

Comment:
´This console.log should be removed.   Otherwise the console is cluttered with error-logs if you visit the cockpit as reader:

![image](https://github.com/d-fine/Dataland/assets/99255296/eb17635b-277a-4edc-941c-d19981bf5494)

Reply:
removed